src/components/Timeline/PTimeline.vue

```diff
@@ -49,48 +52,48 @@
   import { kebabCase } from '@/utilities/strings'
 
   const props = defineProps<{
-    items: TimelineItem[],
-    itemKey?: string,
+    items: T[],
+    itemKey?: K,
     layout?: TimelineLayout | TimelineLayoutFunction,
   }>()
 
-  const layout = computed(() => props.layout ?? 'date-left')
+  const internalLayout = computed(() => props.layout ?? 'date-left')
```

Comment:
cc @pleek91 I _think_ this was the source of the layout warnings we were looking at